Datadog/Example/Debugging/DebugWebviewViewController.swift

```diff
@@ -0,0 +1,106 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import UIKit
+import Datadog
+import WebKit
+
+class DebugWebviewViewController: UIViewController {
+    @IBOutlet weak var rumServiceNameTextField: UITextField!
+
+    /// When `true`, a native RUM Session between `DebugWebviewViewController` and `WebviewViewController` will be tracked.
+    private var useNativeRUMSession = false
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        rumServiceNameTextField.text = (appConfiguration as? ExampleAppConfiguration)?.serviceName
+
+        webviewURLTextField.placeholder = webviewURL
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        if useNativeRUMSession {
+            Global.rum.startView(viewController: self)
+        }
+    }
+
+    override func viewDidDisappear(_ animated: Bool) {
+        if useNativeRUMSession {
+            Global.rum.stopView(viewController: self)
+        }
+    }
+
+    // MARK: - Starting native RUM session
+
+    @IBAction func didTapStartNativeRUMSession(_ sender: Any) {
+        useNativeRUMSession = true
+        Global.rum.startView(viewController: self)
+    }
+
+    // MARK: - Starting webview
+
+    @IBOutlet weak var webviewURLTextField: UITextField!
+
+    private var webviewURL: String {
+        webviewURLTextField.text!.isEmpty ? "https://www.datadoghq.com" : webviewURLTextField.text!
+    }
```

Comment:
/nit to avoid any crash

```suggestion
    private var webviewURL: String {
    guard
        let text = webviewURLTextField.text,
        !text.isEmpty 
    else { return  "https://www.datadoghq.com" }
    
    return text
    }
```